README.md

```diff
@@ -627,6 +627,26 @@ Content-Type: application/xml
 {{(verb-part)}}
 ```
 
+In HTTP, the end-of-line marker is CRLF(`\r\n`) instead of LF(`\n`). Although some web servers handle LF as CRLF for compatibility, some do not. If you encounter a similar problem, try to insert CR(`\r`) manually. For example:
+
```

Comment:
```suggestion
In HTTP, the end-of-line marker is CRLF (`\r\n`) instead of LF (`\n`). Although some web servers handle LF as CRLF for compatibility, some do not. If you encounter a similar problem, try to insert CR (`\r`) manually. For example:

```